lib/CheckboxItem/CheckboxItemAccessibilitySupport.js

```diff
@@ -32,7 +32,7 @@ module.exports = {
 			this.setAttribute('role', enabled ? 'checkbox' : null);
 			this.setAttribute('tabindex', enabled ? 0 : null);
 			this.setAttribute('aria-checked', enabled ? String(this.checked) : null);
-			this.setAttribute('aria-labelledby', enabled ? this.$.client.getId() : null);
+			this.setAttribute('aria-labelledby', !this.accessibilityLabel && enabled ? this.$.client.getId() : null);
```

Comment:
CheckboxItem also has aria-labelledby attribute because It extends Item, so If user add accessibilityLabel to CheckboxItem, CheckboxItem should not have aria-labelledby attribute for reading accessibilityLabel text